webview/src/shared/components/icons/GitCommit.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react'
+import { SVGProps } from 'react'
+
+const SvgGitCommit = (props: SVGProps<SVGSVGElement>) => (
```

Comment:
I'm planning to add the `git-commit` icon in the experiment table as well but doing it in a separate pr since it's taking a little longer than expected.